dnssec.go

```diff
@@ -542,20 +542,20 @@ func (k *DNSKEY) publicKeyRSA() *rsa.PublicKey {
 		explen = uint16(keybuf[1])<<8 | uint16(keybuf[2])
 		keyoff = 3
 	}
+	if explen > 4 {
+		// Larger exponent than supported by the crypto package.
```

Comment:
What does the spec say about leading zeros and what do other implementations do? Are they allowed or prohibited?

Reply:
Leading zeros aren't allowed. I'm not sure what any other implementations do other than my own which disallows leading zeros.